modules/ROOT/pages/oauth-grant-types-about.adoc

```diff
@@ -3,14 +3,14 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-OAuth 2.0 specify the following methods (ie Grant Types) for requesting for a token.
+OAuth 2.0 provides the following grant types methods for requesting for a token:
```

Comment:
"specifies" is probably the right word. OAuth 2.0 is just a spec and every software vendor implements it.

Reply:
Done! Thanks!